requirements.sh

```diff
@@ -43,8 +43,18 @@ if [ ! -d $directory/freeopcua ]; then
 		mv CMakeLists.txt.$$ CMakeLists.txt
 	cd build
 	echo Installing boost components
-	sudo apt install -y libboost-filesystem-dev
-	sudo apt install -y libboost-program-options-dev
+	which apt >/dev/null 2>&1
+	if [ $? -eq 0 ]; then
+		sudo apt install -y libboost-filesystem-dev
+		sudo apt install -y libboost-program-options-dev
+	else
+		which yum >/dev/null 2>&1
+		if [ $? -eq 0 ]; then
+			sudo yum install -y libboost-filesystem
+			sudo yum install -y libboost-program-options
```

Comment:
Do we need these libraries? As while compiling I see those packages are not available for RHEL
```
Installing boost components
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos
No package libboost-filesystem available.
Error: Nothing to do
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos
No package libboost-program-options available.
Error: Nothing to do
```
And Now make is compiled successfully

Reply:
Yes, but they are also pulled in by other packages, so you probably already have them.